### PR TITLE
demo

### PR DESCRIPTION
**Pull request by mengqiy**

### CODE REVIEW

File: create-pdb/fn-config.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+apiVersion: fn.kpt.dev/v1alpha1
```
Comment (frankfarzan) [inline create-pdb/fn-config.yaml:1]: Eventually this can become an example package in the catalog?

**Review by frankfarzan — COMMENTED** [review]
Cool!

File: inject-sidecar/fn-config.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+apiVersion: fn.kpt.dev/v1alpha1
+kind: StarlarkRun
+metadata:
+  name: inject-sidecar-to-depl
+  annotations:
+    config.k8s.io/function: |
```
Comment (droot) [inline inject-sidecar/fn-config.yaml:6]: I am assuming this annotation will be optional once we move to `kpt fn eval` implementation.

File: create-pdb/fn-config.yaml (hunk 1 above)
Comment (mengqiy) [reply create-pdb/fn-config.yaml:1]: Yes, the 3 directories will become 3 example packages for starlark.

File: inject-sidecar/fn-config.yaml (hunk 2 above)
Comment (mengqiy) [reply inject-sidecar/fn-config.yaml:6]: It seems we are going to have `kpt fn eval --fn-config`. If so, we will no longer need this annotation.

**Pull request merged**